doc/commands.dox

```diff
@@ -4099,18 +4099,20 @@ class Receiver
   because doxygen uses it to detect Javadoc commands.
 
 <hr>
-\section cmdtilde \\~[LanguageId]
+\section cmdtilde \\~[LanguageId] or \\~'{'LanguageId [,LanguageId]*'}'
```

Comment:
I have some concerns.

1) Is there a use-case for this? I can only think about using it as fallback telling some translation is missing.
2) I think the `\~lang` and `\~{lang}` look a bit too similar, and it is not intuitively clear one is the negation of the other, maybe `\~!{lang}` would be more clear.
3) From a usability point of view, I would prefer something like `\~dutch Nederlands \~german Deutsch \~else Fallback to e.g. english text \~`.

Reply:
1. No there is not a direct use case, but it is a bit strange that one cannot easily signal the fallback telling some translation is missing.
2. They are indeed quite similar, `\~!{lang}` would indeed be more clear.
3. I thought about this as well but decided against it as
   - one has to keep track of which language have already been selected and reset the list when `\~` is given / end of a comment block
   - the current solution gives also a possibility to have `\~dutch Nederlands \~german Deutsch \~{dutch} some "Fallback" even for a previous handled language (probably quite unusual / uncommon but gives more flexibility) \~`.

I think best is to implement the solution mentioned under 2.